pkg/kv/kvserver/concurrency/BUILD.bazel

```diff
@@ -8,7 +9,7 @@ go_library(
         "latch_manager.go",
         "lock_table.go",
         "lock_table_waiter.go",
-        "lockstate_interval_btree.go",
+        ":lockstate_interval_btree.go",  # keep
```

Comment:
Out of curiosity, why did you add the colon up front?

Also, are all `# keep` comments manually written, or are they ever generated by gazelle?

Reply:
I don't think the colon up front is strictly necessary, but to my eye it makes it more clear the file is generated inside this package rather than being a source file (like all the other files in this list).

The `# keep` comments themselves are generally manually written. As a rule they're added because otherwise Gazelle would delete the line in question.